src/plugins/notebook/components/Notebook.vue

```diff
@@ -226,13 +226,13 @@ export default {
         createNotebookStorageObject() {
             const notebookMeta = {
                 name: this.internalDomainObject.name,
-                identifier: this.internalDomainObject.identifier
+                identifier: this.internalDomainObject.identifier,
```

Comment:
We only need one of these, not both. My preference would be the `identifier`.

Reply:
updated.